lib/Rest/GeneralRequest.cpp

```diff
@@ -234,8 +234,8 @@ void GeneralRequest::addSuffix(std::string&& part) {
 namespace arangodb {
   template <>
   bool GeneralRequest::parsedValue(std::string const& key, bool valueNotFound) {
-    bool found = false;
-    std::string const& val = value(key, found);
+    bool found;
```

Comment:
valgrind usually complains about uninitialized values even though they may be initialized inside the function call